ts/dv360/src/types.ts

```diff
@@ -173,3 +177,22 @@ export interface Accessors {
   insertionOrders: typeof InsertionOrders;
   lineItems: typeof LineItems;
 }
+
+/**
+ * A map of an ID to a resource.
+ */
+export interface EntityMap<Resource extends DisplayVideoResource> {
+  [entityId: string]: Resource;
```

Comment:
I know we've discussed this before and we're long past the use of this pattern here, but I would like to still point out [this section](https://google.github.io/styleguide/tsguide.html#mapped-conditional-types:~:text=type%20operators%20require%20the%20reader%20to%20mentally%20evaluate%20the%20type%20expression.%20This%20can%20make%20programs%20substantially%20harder%20to%20read) of the style guide, which I feel is still applicable here. Object.values() feels like a code smell, and is obfuscates understanding the type. But again, we are well past that here, so I won't let that be a blocker.

Reply:
No, you're right. It's better to clean up if we see it too.

Is this OK? It really doesn't show up much.